core-bundle/src/Cron/CronJob.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+final class CronJob
```

Comment:
We have not yet agreed on the concept of final classes, therefore the `final` keyword has to be removed (unless there are technical reasons that require it).

Reply:
But it is used here too: https://github.com/contao/contao/blob/133696aa02ef2dfb4005937e25b8c8427de885ec/core-bundle/src/Migration/MigrationResult.php#L15

Should it get removed from both?

Reply:
Yes, until we have agreed on a concept, it needs to be removed there as well. Right now, we are only using final classes for

* Anything that `implements ServiceTagInterface`
* Any `*Events` class
* The `ContaoModuleBundle` class
* Anything that `extends AbstractFragmentAnnotation`